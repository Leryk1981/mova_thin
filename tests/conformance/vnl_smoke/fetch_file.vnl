fetch from="https://api.example.com/x" headers.api_key="{{API_KEY}}" rate_limit=10

