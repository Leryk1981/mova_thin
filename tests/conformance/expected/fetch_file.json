{
  "mova_version": "3.3",
  "id": "fetch_file",
  "description": "Generated from VNL",
  "policies": {
    "timeout_ms": 15000
  },
  "actions": [
    {
      "type": "http_fetch",
      "endpoint": "https://api.example.com/x",
      "method": "GET"
    }
  ]
}